bluesky/suspenders.py

```diff
@@ -525,3 +525,123 @@ def _get_justification(self):
                 )
         return ': '.join(s for s in (just, self._tripped_message)
                          if s)
+
+
+class SuspendWhenChanged(SuspenderBase):
+    """
+    Suspend when the monitored value deviates from the expected.
+    
+    Only resume if allowed AND when monitored equals expected.
+    
+    .. note: this suspender is designed to require bluesky restart if value changes
+    
+    USE CASE:
+    
+    :class:`~SuspendWhenChanged()` is useful when ``signal`` is an EPICS enumeration 
+    ([``mbbo`` record](https://wiki-ext.aps.anl.gov/epics/index.php/RRM_3-14_Multi-Bit_Binary_Output))
+    used with a multi-instrument facility.  
+    Choices are the instruments allowed to control any shared hardware.
+    The ``signal``, set by instrument staff outside of bluesky,
+    names which instrument is allowed to control the hardware.
+    Other instruments not matching ``signal`` are expected **not** to 
+    control the hardware (they could use simulators instead or not operate
+    the shared hardware).
+    
+    Since a decision of hardware *vs.* simulators is made at the 
+    time a bluesky session starts and ophyd objects are first created, the 
+    session needs to be aware immediately if the ``signal`` is changed.  
+    The default value of ``allow_resume=False`` defends this decision.
+    If there is a mechanism engineered to toggle ophyd signals between 
+    hardware and simulators, one might consider ``allow_resume=False``.
+    
+    USAGE::
```

Comment:
```suggestion
    Examples
    --------
```

Reply:
Oh, I suggested to use the code-block instead, but they can be actually combined 🙂